quartz/components/Head.tsx

```diff
@@ -1,11 +1,12 @@
+import i18next from "i18next"
 import { FullSlug, _stripSlashes, joinSegments, pathToRoot } from "../util/path"
 import { JSResourceToScriptElement } from "../util/resources"
 import { QuartzComponentConstructor, QuartzComponentProps } from "./types"
 
 export default (() => {
   function Head({ cfg, fileData, externalResources }: QuartzComponentProps) {
-    const title = fileData.frontmatter?.title ?? "Untitled"
-    const description = fileData.description?.trim() ?? "No description provided"
+    const title = fileData.frontmatter?.title ?? i18next.t("head.untitled")
+    const description = fileData.description?.trim() ?? i18next.t("head.noDescriptionProvided")
```

Comment:
should we also add options to parse i18n from frontmatter as well?

I imagine users writing their own file would be a lot nicer to do something like this

```markdown
---
title: chaos is a state of mind.
i18n:
  fr: le chaos est un état d'esprit
  es: el caos es un estado mental
---
```

Users can also set a default language in `quartz.config.ts` as well.

Reply:
edit: shoudl we make this into a plugin instead?. We can probably parse different h3 nodes and others part from a transformers standpoint as well, instead of manually doing this.

I wonder this would get out of hand pretty fast as well. But I don't have an opinion on this other than maintainability of quartz.

Reply:
probably a bit also out of scope, but would be nice to have a way to change locale as well 🤔

Reply:
I think we should split. If this PR is for  "quartz system" (interface), we could have another PR as a plugin to localize user content

Reply:
yes that makes sense. Feel free to ping me I can help with the review.